Modules/_lzmamodule.c

```diff
@@ -755,6 +798,9 @@ _lzma.LZMACompressor.__new__
         have an entry for "id" indicating the ID of the filter, plus
         additional entries for options to the filter.
 
+    threads: int(c_default="1") = 1
+        Number of threads to use for compression (only relevant with FORMAT_XZ).
```

Comment:
Would it be better to keep the Python API unchanged, and implicitly use the cores available?

Reply:
Hmm. This would usually be an improvement. There is a little bit of risk in the way of someone running multiple instances in parallel...

Reply:
IIRC, yes, it's possible to use the chunked/parallel compressor with one thread. xz 5.6.0 will default to this for its command line interface at least, because it enables _decompression_ in parallel for the archives.

Reply:
See also:

- https://github.com/tukaani-project/xz/issues/89#issuecomment-1967105982